inventory/byo/hosts.origin.example

```diff
@@ -530,8 +530,8 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # list of options please see roles/openshift_logging/README.md
 #
 # Configure loggingPublicURL in the master config for aggregate logging, defaults
-# to https://kibana.{{ openshift_master_default_subdomain }}
-#openshift_hosted_logging_hostname=https://kibana.example.com
+# to kibana.{{ openshift.common.dns_domain }}
```

Comment:
that is what we use in the openshift_logging role though. 
from defaults/main.yaml
`openshift_logging_kibana_hostname: "{{ openshift_hosted_logging_hostname | default('kibana.{{openshift.common.dns_domain}}') }}"`

Its in the case where they are using the playbooks/byo/openshift-cluster/openshift-logging.yaml playbook. That may not be set